content/docs/1.6.0/deploy/install/install-with-flux.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Install with Flux
+weight: 11
+---
+
+## Prerequisites
+- Your workstation: Install [Helm](https://helm.sh/docs/) v3.0 or later.
+- Kubernetes cluster:
+  - Ensure that each node fulfills the [installation requirements](../#installation-requirements).
+  - Follow the [official document](https://fluxcd.io/flux/installation/#install-the-flux-cli) to install Flux CLI.
```

Comment:
```suggestion
  - [Install the Flux CLI](https://fluxcd.io/flux/installation/#install-the-flux-cli).
  - [Bootstrap Flux with GitHub](https://fluxcd.io/flux/installation/bootstrap/github/) using the Flux CLI.

    Run the following commands to export your GitHub personal access token (PAT) as an environment variable, deploy the Flux controllers on your cluster, and configure the controllers to sync the cluster state from the specified GitHub repository.

    ```bash
    export GITHUB_TOKEN=<gh-token>
    flux bootstrap github \
      --token-auth \
      --owner=<github_username> \
      --repository=<github_repo_name> \
      --branch=<branch_name> \
      --path=<folder_path_within_repo> \
      --personal
    ```
```

Reply:
I moved the bootstrap info here and expanded the code block intro to improve the procedure's overall flow.